docs/querying/basics.md

```diff
@@ -92,8 +92,13 @@ single sample value for each at a given timestamp (instant): in the simplest
 form, only a metric name is specified. This results in an instant vector
 containing elements for all time series that have this metric name.
 
+The value returned will be the most recent sample within the the time frame
```

Comment:
There's a repeated "the the" here.

But besides that, "within the time frame specified by the API call" is a bit unclear or not quite correct. For example if you have a range query over a 1h time frame, it's not a single most recent value within that time frame that is being selected, but the most recent sample value found within a 5-minute lookback window from each individual resolution step. Or in the case of an instant query, it is the most recent one looking back from the single evaluation timestamp.

How about this instead:

"The value returned will be that of the most recent sample prior to the query's evaluation timestamp (in the case of an [instant query](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries)) or of the current resolution step within the query (in the case of a [range query](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries)). The [`@` modifier](#modifier) allows overriding the timestamp relative to which the selection takes place. Time series where the most recent sample is older than the [staleness threshold](#staleness) will not be returned."